src/components/detailedProduct/detailedProductView.ts

```diff
@@ -1,52 +1,37 @@
 import HTMLCreator from '../HTMLCreator';
-import slider1 from '../../assets/slider-1.jpg';
-import slider2 from '../../assets/slider-2.jpg';
-import slider3 from '../../assets/slider-3.jpg';
-import slider4 from '../../assets/slider-4.jpg';
-import slider5 from '../../assets/slider-5.jpg';
-import slider6 from '../../assets/slider-6.jpg';
+import DetailedProductController from './detailedProductController';
 
 export default class DetailedProduct {
-  name: string = 'Product';
+  controller: DetailedProductController;
 
-  price: string = 'Price';
+  id: string;
 
-  discountPrice: string = 'Discount Price';
-
-  descriptionOfProduct: string = 'Description';
+  constructor(id: string) {
+    this.controller = new DetailedProductController();
+    this.id = id;
+  }
 
   renderMain() {
     const detailedProductWrapper = HTMLCreator.createElement('div', { class: 'detailed__product-wrapper' });
     const main = HTMLCreator.createElement('main', { class: 'detailed__product' }, [
       HTMLCreator.createElement('div', { class: 'detailed__product-container' }, [detailedProductWrapper]),
     ]);
```

Comment:
The method renderMain is constructing HTML elements effectively. Consider breaking down the creation of complex elements into smaller helper methods to improve readability.